src/core/lib/support/wrap_memcpy.c

```diff
@@ -40,7 +40,7 @@
  */
 
 #ifdef __linux__
-#ifdef __x86_64__
+#if defined(__x86_64__) && defined(__GNU_LIBRARY__)
```

Comment:
So the right thing to do here would rather to fully remove that case, and always falling back on memmove below. The reason is that pre-compiled binaries from Linux would then run on Alpine, whereas this would force them to be compiled there instead.

Unfortunately, the other problem above (with sched_getcpu) would still be present, and I don't have a good alternative for it.